packages/cfstream/cfstream.1.3.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+homepage: "https://github.com/biocaml/cfstream"
+dev-repo: "https://github.com/biocaml/cfstream.git"
+bug-reports: "https://github.com/biocaml/cfstream/issues"
+license: "LGPL + linking exception"
+maintainer: "Ashish Agarwal <agarwal1975@gmail.com>"
+authors: [
+  "Philippe Veber <philippe.veber@gmail.com>"
+  "Ashish Agarwal <agarwal1975@gmail.com>"
+  "Drup <drupyog@zoho.com>"
+]
+
+build: [
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "jbuilder" {build}
+  "core_kernel" {>= "v0.11.0"}
+  "ounit" {test}
```

Comment:
This dependency is unnecessary as there is no `build-test` phase.

Reply:
I deleted this.